packages/oruga-next/src/components/table/Table.vue

```diff
@@ -135,6 +135,39 @@
                     </th>
                     <th v-if="checkable && checkboxPosition === 'right'" />
                 </tr>
+                <tr v-if="hasCustomSubheadings" class="o-table__th">
+                    <th v-if="showDetailRowIcon" width="40px" />
+                    <th v-if="checkable && checkboxPosition === 'left'" />
+                    <th
+                        v-for="(column, index) in visibleColumns"
+                        :key="column.newKey + ':' + index + 'subheading'"
+                        :style="column.style"
+                    >
+                        <div
+                            :class="{
+                                'is-numeric': column.numeric,
```

Comment:
`numeric` is not relative to a specific style at the moment (you can remove it) and `centered` doesn't exist, we use a generic position prop that you can move on above `th` (you might convert the supposed `thSubheadingClasses` from computed to a method) and remove the `div`

Reply:
@jtommy I believe i got this done correctly based off your feedback, but am still uncertain. Let me know if I need to make a change here after you view the new changes.